### PR TITLE
Footer Added

### PR DESCRIPTION
**Pull request by Rasheduluctit**
Added The Footer and It is responsive.

### CODE REVIEW

File: src/components/Footer/FooterGridItem.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { GridItem, Text } from "@chakra-ui/react";
+import React from "react";
+
+const FooterGridItem = ({ column }: { column: any }) => {
+  return (
+    <>
```
Comment (saminsss) [inline src/components/Footer/FooterGridItem.tsx:6]: React.Fragment

File: src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -1,11 +1,104 @@
...
+              fontWeight="normal"
+            >
+              {/* Take a Quick Tour */}
+              {footerData?.footerBtn2}
+            </Button>
+          </Flex>
+        </Box>
+        <Divider orientation="horizontal" my="20px"></Divider>
+        <Grid
+          templateColumns={{ base: "null", md: "repeat(5,1fr)" }}
```
Comment (saminsss) [inline src/components/Footer/Footer.tsx:65]: Why is `base` null

File: src/components/Footer/Footer.tsx [hunk 3]
```diff
@@ -1,11 +1,104 @@
...
+          justifyContent="space-between"
+          py="30px"
+          flexDirection={{ base: "column", md: "row" }}
+        >
+          <Text>
+            {/* © Teachworks 2021 */}
+            {footerData?.footerCopyRight}
+          </Text>
+          <Box mr="200px">
+            <Button
```
Comment (saminsss) [inline src/components/Footer/Footer.tsx:82]: Button should navigate to social links

https://www.facebook.com/uttaracomputertraininganditfirm

File: src/components/Footer/Footer.tsx [hunk 4]
```diff
@@ -1,11 +1,104 @@
...
+              borderRadius="50%"
+              color="secondary"
+              bg="transparent"
+              _hover={{ bg: "secondary", color: "black" }}
+            >
+              <Icon as={ImFacebook} />
+            </Button>
+            <Button
```
Comment (saminsss) [inline src/components/Footer/Footer.tsx:90]: Twitter button not required

**Review by saminsss — CHANGES_REQUESTED** [review]

File: src/components/Footer/Footer.tsx (hunk 2 above)
Comment (Rasheduluctit) [reply src/components/Footer/Footer.tsx:65]: For mobile devices, the grid should behave as it is. No condition is required for the base devices and that's why it is null. For medium to large devices it should divide my content box into five columns.

File: src/components/Footer/Footer.tsx (hunk 3 above)
Comment (Rasheduluctit) [reply src/components/Footer/Footer.tsx:82]: done

File: src/components/Footer/Footer.tsx (hunk 4 above)
Comment (Rasheduluctit) [reply src/components/Footer/Footer.tsx:90]: done

File: src/components/Footer/FooterGridItem.tsx (hunk 1 above)
Comment (Rasheduluctit) [reply src/components/Footer/FooterGridItem.tsx:6]: done

**Comment by Rasheduluctit** [comment]
[footer.webm](https://github.com/UCT-IT/edume/assets/149924448/3d69c538-31bb-4fe5-9867-267bfc6bceb9)

**Review by saminsss — APPROVED** [review]
Looks good!

**Pull request merged**